### PR TITLE
Updating documentation

### PR DESCRIPTION
**Pull request by Oluane**
## Description

Small update of the README documentation.

## Type of change

- [x] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [ ] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## Checklist

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

**Comment by frieda-ewlines** [comment]
Task linked: [CU-ar3fq1 [chore] - Updating documentation](https://app.clickup.com/t/ar3fq1)

File: README.adoc [hunk 1]
```diff
@@ -330,8 +348,11 @@ Favicons for various OS.
 - *configs/*: 
   * *config-variables.ts*: Entry point used to verify env vars sourcing, and prevent the app to run if forgotten.
   * *db-client.ts*: Singleton of the DynamoDB client.
+  * *intl.ts*:
```
Comment (Oluane) [inline README.adoc:351]: @Fuzznimp 😬

Comment (Fuzznimp) [reply README.adoc:351]: > File regrouping all the `intl` object, which are instances to store the cache of all Intl.* APIs, configurations, compiled messages and such.

File: README.adoc [hunk 2]
```diff
@@ -682,17 +705,17 @@ We are using https://github.com/testing-library/dom-testing-library[Testing Libr
...
-Since we had to trigger different flows based on a third party event (i.e. `review app deployment`), we couldn't leverage Github checks management automation on PR.
-Once we receive the status of the deployment from our third party actor, we run our end to end test suite (`e2e-tests` job inside `e2e-test.yml`) if the review app deployment is a success, or we skip it in the case of a failure during the process.
+Several flows (or `jobs`) are executed in this context and orchestrated in workflows inside the `config.yml` file: 
 
-All of these cases are handled at job-level, mostly thanks to https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions[job status check functions], which permit to control more precisely jobs execution via conditional statements. 
+- *lint-and-tests*: runs code linting and integration/unit tests.
+- *e2e-tests*: runs e2e tests suite on a deployed Connect Account version.
+- *lighthouse*: runs Lighthouse reports on a deployed Connect Account version.
+- *promote-to-production*: use to automate production deployment from staging when all tests suite is successful.
```
Comment (Fuzznimp) [inline README.adoc:717]:
```suggestion
- *lint-and-tests*: runs the code linting and the integration/unit tests.
- *e2e-tests*: runs the e2e test suite on a deployed Connect Account version.
- *lighthouse*: runs the Lighthouse audits on a deployed Connect Account version.
- *promote-to-production*: used to automate the production deployment from staging when the test suite is successful.
```

File: README.adoc [hunk 3]
```diff
@@ -360,8 +381,10 @@ Favicons for various OS.
 - *app.json*: Building instructions for Heroku.
 - *assets.d.ts*: Type declaration allowing the import of assets in TypeScript files.
 - *Brewfile*: Tools needed that will be installed via Brew for MacOS users.
+- *connect-profile-openapi.yml*: OpenAPI file of Connect Profile.
 - *docker-compose.yml*: Instructions to launch DynamoDB and the observability tools.
 - *Dockerfile*: Instructions for Docker image build process.
+- *lighthouserc.js*: Instructions for Lighthouse reviews.
```
Comment (Fuzznimp) [inline README.adoc:387]:
```suggestion
- *Dockerfile*: Instructions for Docker image build process.
- *lighthouserc.js*: Instructions for Lighthouse audits.
```

File: README.adoc [hunk 4]
```diff
@@ -263,18 +263,36 @@ if you want to automatically fix linting issues.
...
 
 | `CONNECT_TEST_ACCOUNT_PASSWORD`
 | Password of the Connect account that will be used for e2e tests.
 
+| `CONNECT_TEST_ACCOUNT_SUB`
+| Sub of the Connect account that will be used for e2e tests.
+
+| `CONNECT_TEST_ACCOUNT_URL`
+| URL of the Connect Account application that will be used for e2e tests.
+
```
Comment (Fuzznimp) [inline README.adoc:295]: Could you renamed the `Connect account` mentions with `Connect.Account`? 🙇‍♂️.

**Review by Fuzznimp — APPROVED** [review]

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 1</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.87</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.84</li><li>Accessibility: 0.91</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.92</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Comment by frieda-ewlines** [comment]
<details><summary>Click here to see the Lighthouse manifest</summary><br><p><strong>/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.98</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/:</strong></p><ul><li>Performance: 0.95</li><li>Accessibility: 0.89</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/:</strong></p><ul><li>Performance: 0.9</li><li>Accessibility: 0.89</li><li>Best practice: 0.93</li><li>SEO: 1</li></ul><hr><p><strong>/account/profile/user-profile/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/:</strong></p><ul><li>Performance: 0.94</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/email/new/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/logins/phone/new/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/:</strong></p><ul><li>Performance: 0.91</li><li>Accessibility: 0.88</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/security/sudo/:</strong></p><ul><li>Performance: 0.89</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr><p><strong>/account/locale/:</strong></p><ul><li>Performance: 0.88</li><li>Accessibility: 0.9</li><li>Best practice: 1</li><li>SEO: 1</li></ul><hr></details>

**Pull request merged**